news/index.md

```diff
@@ -1,139 +1,14 @@
 ---
-layout: page
-title:  "News"
-menu_item: true
```

Comment:
Why this removal?
I might misunderstood things, but I think `menu_item: true` is needed so that "news" item appears selected on the top menu.

Reply:
uhm, that's my fault, I did it because I thought it was handled differently once code changed, I never saw that setting. Once built the site I had no problems so I forgot to add it back. Strange, because if the pages are rebuilt the menu had to disappear, or maybe my browser cached it.